### ISSUE TITLE
gsalib not available

### ISSUE THREAD

**Issue by jonahcullen** [broadinstitute/gsalib#3]
Hello, I have a singularity container that utilizes `gsalib` with R version 3.6.3. As of about a week ago, building the container worked without issue. Yesterday during a rebuild, I was unable to install `gsalib` with 

```
Warning message:
package 'gsalib' is not available (for R version 3.6.3)
```

Did something perhaps change on CRAN? I also tried to use R version 4.2.1 in the container but was foiled as `gsalib` is not available for that version either.

Thank you for your time!

**Comment by lbergelson** [comment]
Hi @jonahcullen,

CRAN decided to archive the gsalib package because R change out from under it in the many years since it was last updated.  We're in the process of getting it reinstated but the entire CRAN team also went on vacation and closed submissions at the same time. It should hopefully be back up soon.  

It's unclear if the old release will be available again or if you will have to switch to a new version: 2.2.1.

Alternatively, you should be able to install it by downloading from the archive (https://cran.r-project.org/src/contrib/Archive/gsalib/gsalib_2.1.tar.gz) and installing from the tar.

**Comment by lbergelson** [comment]
This is fixed now, a new version, 2.2.1 is now available again on CRAN.  Sorry for the long delay.
